jep/200/README.adoc

```diff
@@ -385,10 +383,25 @@ include links to the pull requests and mailing list discussions which were invol
 
 == Appendix A. Tested Plugins
 
-<!-- TODO: Explicitly covered what was covered...
+Current state:
 
-* Everything in ATH
-* PCT for a set of plugins, @jglick and @oleg-nenashev will construct a list. Generally, "compatible" and "verified" plugins from https://go.cloudbees.com/plugins/
-* @oleg-nenashev will review the uncovered plugins, and then we will plan testing for them before the JEP changes get released
+* link:https://github.com/jenkinsci/acceptance-test-harness[Jenkins Acceptance Test Harness] has been executed with the patched components, several plugins were fixed (see the _Testing_ section)
+* link:https://github.com/jenkinsci/plugin-compat-tester[Plugin Compatibility Tester (PCT)] has been executed for a limited plugin set
+** Not all plugins recommended in the Setup Wizard have been verified
+** Currently the Jenkins project currently has no maintained CI job running PCT against all plugins suggested by the Setup Wizard
+** The list of tested plugins is provided below
 
--->
+// TODO: Explicitly covered what was covered...
+//
+//* Everything in ATH
+//* PCT for a set of plugins, @jglick and @oleg-nenashev will construct a list. Generally, "compatible" and "verified" plugins from https://go.cloudbees.com/plugins/
+//* @oleg-nenashev will review the uncovered plugins, and then we will plan testing for them before the JEP changes get released
+//
+
+=== Plugins verified by PCT
+
+NOTE: The section is in progress
+
+|=========================================================
+| Plugin | Issues found? | Released in | Issue links
```

Comment:
If there are any violations found, I will just immediately update the core PR and (if appropriate) file a matching plugin PR. No need for issue links.

Reply:
The main idea is to track examples of fixes for the developers who may need examples + to track info what discovers the issue: ATH/PCT/manual exploratory testing